src/app/app.module.ts

```diff
@@ -138,6 +140,8 @@ import { DashboardShortcutWidgetModule } from "./core/dashboard-shortcut-widget/
       ...DemoUserGeneratorService.provider(),
     ]),
     AttendanceModule,
+    MatChipsModule,
```

Comment:
Shouldn't we import the `EntitySelectModule` here instead of adding Material modules?